docs/rst/user_manual/configuration.rst

```diff
@@ -387,6 +387,54 @@ QoS
 
     The :ref:`Topic QoS <user_manual_configuration_topic_qos>` configured in ``specs`` can be overwritten by the :ref:`Participant Topic QoS <user_manual_configuration_participant_topic_qos>` and the :ref:`Manual Topics <user_manual_configuration_manual_topics>`.
 
+.. _router_specs_logging:
+
+Logging
```

Comment:
Please include a note saying that for the logs to work, the -DLOG_INFO=ON compilation flag is required.